namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-tier-dev/resources/rds.tf

```diff
@@ -0,0 +1,63 @@
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ */
+
+variable "cluster_name" {
+}
+
+variable "cluster_state_bucket" {
+}
+
+module "hmpps_tier_dev_rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=5.12"
+
+  cluster_name           = var.cluster_name
+  cluster_state_bucket   = var.cluster_state_bucket
+  team_name              = "manage-a-sentence"
+  business-unit          = "HMPPS"
+  application            = "manage-a-sentence"
+  is-production          = "false"
+  namespace              = var.namespace
+  environment-name       = "development"
+  infrastructure-support = "hmpps@digital.justice.gov.uk"
+  rds_family             = "postgres10"
+  db_parameter           = [{ name = "rds.force_ssl", value = "1", apply_method = "immediate" }]
+
+  providers = {
+    aws = aws.london
+  }
+}
+
+resource "kubernetes_secret" "hmpps_tier_rds_secrets" {
+  metadata {
+    name      = "hmpps-tier-rds"
+    namespace = "hmpps-tier-dev"
```

Comment:
If you use `var.namespace` here (as per line 21), you never have to worry about copy/paste errors, or changing something in one place but not another, by mistake.